### ISSUE TITLE
An elaborate explanation of the APISIX Architecture.

### ISSUE THREAD

**Issue by Anita-ihuman** [iresty/iresty.github.io#867]
**Actual Behaviour**
The graphical representation of the architecture does not comprehensively explain how the architecture works.
 

**Expected Behaviour**
Put the explanation of this architecture into words.
To help users understand the projects functionalities.
 

**Steps to reproduce it**

 

**Screenshots of the issue**
![Screenshot 2022-02-04 at 09 38 24](https://user-images.githubusercontent.com/62384659/152498284-86e59118-f8b4-4d52-8160-8ca8f29e5531.png)

 

**Would you like to work on the issue?**

**Comment by juzhiyuan** [comment]
Hi @Anita-ihuman, you could ask help from @membphis IMO :)

**Comment by membphis** [comment]
@bzp2010 @shuaijinchao @tzssangglass Would you like to fix this issue?

**Comment by Anita-ihuman** [comment]
@bzp2010 @shuaijinchao @tzssangglass, We can collaborate on this one.

**Comment by tzssangglass** [comment]
Gladly, can we collaborate through mailing lists and PR?

**Comment by bzp2010** [comment]
Hi, @Anita-ihuman. It is a pleasure to have your contribution to this project.

Of course, you can communicate via the mailing list, issue or Apache's Slask channel `#apisix` if you wish. And, what kind of help information do you need?

**Comment by shuaijinchao** [comment]
> @bzp2010 @shuaijinchao @tzssangglass, We can collaborate on this one.

Of course, very willing to complete this PR together. We can communicate via the mailing list or Slack.

**Comment by Anita-ihuman** [comment]
Thank you slack works very well.
Basically, I need an explanation of the architecture that gives a rundown of how APISIX and its features satisfy a user's needs.

**Comment by Anita-ihuman** [comment]
@navendu-pottekkat

**Comment by pottekkat** [comment]
Thank you @Anita-ihuman 

I will create a draft document that I will open for review to make sure that there isn't any wrong information.

I will update in this thread.

**Comment by yzeng25** [comment]
Closing this due to prolonged inactivity.

Since this issue that hasn’t seen updates in a long time to keep the tracker manageable. If this is still relevant, please reopen with up‑to‑date details or open a new issue referencing this one.
